novi-sajt/css/pages/o-klubu.css

```
@@ -0,0 +1,278 @@
+/* About */
+
+.about-section {
+  padding: 40px 20px;
+  background: #0f2538;
+}
+
+.about-wrapper {
+  max-width: 1200px;
+  margin: 0 auto;
+
+  display: grid;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)